frites/conn/conn_tf.py

```diff
@@ -0,0 +1,217 @@
+"""Compute time-frequency decomposition base on Morlet or Multitaper methods.
+
+This script contains the function:
+
+1. _tf_decomp used to decompose the sinal in tf domains using Morlet or
+    Multitaper
+1. _create_kernel: Create a kernel to smooth the spectra (either boxcar or
+    hanning)
+2. _smooth_kernel: Perform the smoothing operation on the spectra based on the
+   convolution theorem
+"""
+# Authors : Vinicius Lima <vinicius.lima.cordeiro@gmail.com >
+#           Etienne Combrisson <e.combrisson@gmail.com>
+#
+# License : BSD (3-clause)
+
+
+import numpy as np
+
+from mne.time_frequency import tfr_array_morlet, tfr_array_multitaper
+from scipy.signal import fftconvolve
+
+from frites.io import logger
+
+
+def _tf_decomp(data, sf, freqs, mode='morlet', n_cycles=7.0, mt_bandwidth=None,
+               decim=1, kw_cwt={}, kw_mt={}, n_jobs=1):
+    """Time-frequency decomposition using MNE-Python.
+    Parameters
```

Comment:
@ViniciusLima94 can you put a space before the "Parameters" line and also before the "Returns" one? Otherwise it will not appears correctly on the doc